### PR TITLE
Add icon for popup action

### PR DESCRIPTION
**Pull request by xuanquocc**
# Description

Add icon and theme for Popup Action

## Detailed PR Verification Procedure(s)
None

## Ticket

[GS-35](https://tito-training.atlassian.net/browse/GS-35)

## Application Checklist

- [x] **JIRA Linked**: Author has followed template and has a title in the following format:
  > JIRA-0: Generic summary of work done in PR
- [ ] **No Breaking Changes**: Author has confirmed all API contract changes are non-breaking.
- [x] **Changes Unit Tested**: Author has unit tested new code to prove it works.
- [ ] **Application Passes All Tests**: Author ensures this pull-request passes `npm test`.
- [x] **Author Self Reviewed**: Author of pull-request has self-reviewed this code.
- [x] **Author is Assignee**: Author has assigned self as assignee of pull request.

## Security Checklist

- [ ] **Code Review for Security:** Developer has self-reviewed this code, based on OWASP Top-10 vulnerabilities and general environment security attributes, to avoid dversely impact the security of the system/environment/application, to the best of their abilities.
- [ ] **Backout Procedures:** Developer has considered how the changes presented in this pull request can be replaced with a prior version of these changes, fixed going forward, and/or implemented to be backwards compatible.

### CODE REVIEW

**Comment by baonguyen26** [comment]
Pls resolve conflict

File: src/assets/theme/index.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ const theme = {
       paper: "#f6f7f8",
       divider: "#e9eaec",
       disabled: "#e8eaed",
+      primary: "#ECF7FD",
```
Comment (Lanng21) [inline src/assets/theme/index.ts]: Existed in file blue.ts

File: src/assets/icons/Ticket.svg
Comment (Lanng21) [inline src/assets/icons/Ticket.svg:1]: why did you change this file?????

File: src/assets/icons/Call.svg
Comment (Lanng21) [inline src/assets/icons/Call.svg:1]: Don't change other people's files!

**Review by duong56 — APPROVED** [review]
LGTM! (Other comment)
